px/main.py

```diff
@@ -598,6 +598,7 @@ def set_username(username):
     State.username = username
 
 def set_password():
+    sys.exit()
```

Comment:
To stick with the styling of commenting out code we don't want, can you comment out everything here up to the original `sys.exit()`?